install/gcp/deployment_manager/istio-cluster.jinja.display

```diff
@@ -0,0 +1,146 @@
+metadataVersion: v1test
+
+description:
+  title: Istio on GKE Launcher
+  version: 0.0.1
+  url: 'https://istio.io/'
+  tagline: Running Istio on GKE
+  descriptionHtml: 'GKE with Istio'
+  logo: 'https://avatars3.githubusercontent.com/u/23534644?s=100&v=4'
+  icon: 'https://avatars3.githubusercontent.com/u/23534644?s=100&v=4'
+  architectureDiagram: 'https://avatars3.githubusercontent.com/u/23534644?s=100&v=4'
+  architectureDescription: 'Istio is an open platform that provides a uniform way to connect, manage, and secure microservices. Istio supports managing traffic flows between microservices, enforcing access policies, and aggregating telemetry data, all without requiring changes to the microservice code'
```

Comment:
Description  supports some basic HTML.  So suggest highlighting that the clusterRole permission is needed:

```
  architectureDescription: 'Istio is an open platform that provides a uniform way to connect, manage, and secure microservices. Istio supports managing traffic flows between microservices, enforcing access policies, and aggregating telemetry data, all without requiring changes to the microservice code. <li><b>Note: You must set your compute service account with "Container Engine Admin" IAM Role.</b></li> <li>For more information, see <a href="https://istio.io/docs/setup/kubernetes/quick-start-gke-dm.html">Quick Start with Google Kubernetes Engine</a></li>'```

Reply:
Updated.